orthoinference/src/main/java/org/reactome/orthoinference/OrthologousEntityGenerator.java

```diff
@@ -189,28 +194,31 @@ private static GKInstance createInfComplexPolymer(GKInstance complexInst, boolea
 			{
 				if ((complexTotalProteinCounts > 0 && complexInferrableProteinCounts == 0) || percent < 75)
 				{
-					logger.info("\tComplex/Polymer protein count is below threshold -- terminating inference");
+					logger.info("Complex/Polymer protein count is below 75% threshold (" + percent + "%) -- terminating inference");
 					return nullInst;
 				}
 			}
-			
+			logger.info("Complex protein counts. Total: " + complexTotalProteinCounts + "  Inferrable: " + complexInferrableProteinCounts);
 			GKInstance infComplexInst = InstanceUtilities.createNewInferredGKInstance(complexInst);
 			infComplexInst.addAttributeValue(summation, complexSummationInst);
 			infComplexInst.addAttributeValue(name, complexInst.getAttributeValue(name));
-			ArrayList<GKInstance> infComponentInstances = new ArrayList<GKInstance>();
+			ArrayList<GKInstance> infComponentInstances = new ArrayList<>();
```

Comment:
One wayward ArrayList that can be changed for List :).

Reply:
Ahh, one squeaked through!

Reply:
LOL it's really not _that_ big of a deal. ;)